customer-samples/AutomaticTraceIdInjection/Log4NetExample/Properties/launchSettings.json

```diff
@@ -1,11 +1,18 @@
 {
   "profiles": {
-    "Log4NetExample": {
+    "LaunchWithServiceUnification": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "DD_LOGS_INJECTION": "true",
+        "DD_ENV": "customer-sample",
+        "DD_VERSION": "1.0.0-beta"
```

Comment:
Do you need `DD_SERVICE` in these settings?

Or even `DD_TAGS=env:,version:service:`?

Reply:
`DD_SERVICE` is usually inferred from the .NET program name, so I was testing that

Reply:
Can we rely on the inference to always succeed?

Our documentation will steer people towards the DD env vars as the source of truth, regardless of the languages/tracers they're using.

Reply:
got it 👍

Reply:
Inference will always succeed in .NET since there's a built-in identity of .NET assemblies, but of course `DD_SERVICE` will work and take precedence.